themes/bootstrap5/templates/myresearch/newpassword.phtml

```diff
@@ -23,6 +23,9 @@
     <input type="hidden" value="<?=$this->escapeHtmlAttr($this->hash) ?>" name="hash">
     <input type="hidden" value="<?=$this->escapeHtmlAttr($this->username) ?>" name="username">
     <input type="hidden" value="<?=$this->escapeHtmlAttr($this->auth_method) ?>" name="auth_method">
+    <?php if (!empty($this->target)): ?>
+      <input type="hidden" value="<?=$this->escapeHtmlAttr($this->target) ?>" name="target">
```

Comment:
Would it make sense to use a more descriptive name here, like `target_ils`? At a glance, it's not immediately obvious to me what target means, so a little context might help.